### DISCUSSION TITLE
An earlier installation failure will result in an IPMI assertion event that remains uncleared even after a successful installation

### DISCUSSION THREAD

**Discussion by prabhakarpujeri** [rhinstaller/anaconda#5069] [General]
for installation Failed  following IPMI command send to iDRAC so overall iDRAC status will be red
0x4 0x1F 0x0 **0x6F**  0xA 0x0 0x0

but until the deassertion event, iDRAC status will not return to green. even if we successfully installed OS next time.

as per IPMI spec Table 32-1  provides an option to Deassertion event with the following IPMI command. which need to be executed when new installation starts 

0x4 0x1F 0x0 **0xEF**  0xA 0x0 0x0

https://www.intel.la/content/dam/www/public/us/en/documents/specification-updates/ipmi-intelligent-platform-mgt-interface-spec-2nd-gen-v2-0-spec-update.pdf

**Comment by VladimirSlavik** [comment]
Thank you for the report. Looks like we always do `0x04 0x1F 0x00 0x6F 0x?? 0x00 0x00` in `ipmi_report` and just swap in a byte from a "table" of events.

I tried to find justification for adding the deassert, but the spec is just too big. Would you know where to find that?

Another point is that this should be in bugzilla, but I can file that once we get all the details...
